README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is easier to read.
+
```

Comment:
the semantic tags actual are indexed differently by screen readers as well as by search engine bots for categorizing and indexing your site on them